src/app/view/components/_common/iframe/iframe-porlet-legacy/iframe-porlet-legacy.component.html

```diff
@@ -1 +1 @@
-<dot-iframe #iframe [src]="url | async"></dot-iframe>
+<dot-iframe #iframe [src]="url | async" [isLoading]="isLoading"></dot-iframe>
```

Comment:
I discuss this with @hmoreras and we think that for now it's the best solution, because the other solution it's to manually manage the ```history state``` of the browser and it could be really messy.